### ISSUE TITLE
[4.3. RC1] Audio block disappears when align right or left in Chrome

### ISSUE THREAD

**Issue by bph** [WordPress/gutenberg#11740]
**Describe the bug**
When I add an audio block to a paragraph block and align it left or right and then click outside the editing space, the audio block disappears. 
It's still there, when I squint I can see the Audio label. . Clicking on it brings back the block into view 

**To Reproduce**
1. Add a new post
2. Add a paragraph with some text
3. add an audio block and align it right
4. click outside the editor space. 

**Expected behavior**
The audio block is still visible 

**Screenshots**
Here is the video. http://recordit.co/898tjKTCao

When I looked closer I see some residue of the block and when I click on it, I get the tools bar and the display of the block back. See in this little video: http://recordit.co/px35d8sUae

**Desktop (please complete the following information):**
Seems to only happen with Google Chrome: 70.0
It works fine with Firefox Developer EDition
63.0b12
Video #3: http://recordit.co/UGsZDzGtu6

In reference of this PR: https://github.com/WordPress/gutenberg/pull/11357

**Comment by dashkevych** [comment]
I've had the same issue when I was adding an audio block in the Editor. I've fixed it by adding this custom CSS to the editor styles:

```
[data-type="core/audio"][data-align="left"] .editor-block-list__block-edit,
[data-type="core/audio"][data-align="right"] .editor-block-list__block-edit {
  width: 50%;
}
```

**Comment by jasmussen** [comment]
Fixed in #11749, thanks for the helpful report.
